test/e2e/app-dir/use-cache-dev/use-cache-dev.test.ts

```diff
@@ -0,0 +1,219 @@
+import { nextTestSetup } from 'e2e-utils'
+import { retry } from 'next-test-utils'
+
+describe('use-cache-dev', () => {
+  const { next, skipped, isNextDev, isTurbopack } = nextTestSetup({
+    files: __dirname,
+    skipDeployment: true,
+  })
+
+  if (skipped) {
+    return
+  }
+
+  if (isNextDev) {
```

Comment:
All tests in this block are unchanged. They just appear as new in the diff because the file was moved from `test/development/app-dir/use-cache-dev/use-cache-dev.test.ts` to `test/e2e/app-dir/use-cache-dev/use-cache-dev.test.ts`.